.circleci/comment.js

```diff
@@ -7,19 +7,57 @@ const octokit = new Octokit({
 run();
 
```

Comment:
Workflow getting a build link on a fork PR:
1. A member comments 'get build link' on fork PR
2. Github action is triggered. Creates a branch based off the fork PR's contents if one doesn't exist. If a branch already exists, updates that branch instead. The commit for this branch is a merge commit based off last `main` commit and the last fork commit.
3. Standard CircleCi commit workflow runs for the branch (untouched in this PR, will need to test if it is triggered with this method of creating/updating branches)
4. CircleCi comment stuff runs. If we detect that the built commit isn't in a PR, we check to see if the commit's parent exists in any PRs. If we find a matching fork PR, comment the build links there